pkgs/by-name/gd/gdu/package.nix

```diff
@@ -43,6 +43,7 @@ buildGoModule rec {
   checkFlags = [
     # https://github.com/dundee/gdu/issues/371
     "-skip=TestStoredAnalyzer"
+    "-skip=TestAnalyzePathWithIgnoring"
```

Comment:
```suggestion
    "-skip=TestStoredAnalyzer,TestAnalyzePathWithIgnoring"
```
do we maybe need to do this?

Reply:
Doesn't seems to work for me.

Reply:
hmm, not sure why it failed in the first place